Portal_Consumer-LOS/Configurations/standard_configuration.json

```diff
@@ -0,0 +1,85 @@
+[
+     {
+          "Sequence": 1,
+          "Object API Name": "clcommon__NavigationStructure__c",
+          "External Id Field": "clcommon__External_ID__c",
+          "label": "NavigationStructure",
+          "Filter Criteria": {
+              "is_root": true,
+              "Where Clause": ""
+          }
+      },
+      {
+          "Sequence": 2,
+          "Object API Name": "clcommon__Actors__c",
+          "External Id Field": "clcommon__External_ID__c",
+          "label": "Actors",
+          "Filter Criteria": {
+              "is_child": true,
+              "First Level Parent Objects": [
+                  {
+                      "Object API Name": "clcommon__NavigationStructure__c",
+                      "Reference Field Name": "clcommon__Navigation_Structure__c",
+                      "is_reference_mandatory": true
+                  }
+              ]
+          }
+      },
+      {
+          "Sequence": 3,
+          "Object API Name": "clcommon__Action__c",
+          "External Id Field": "clcommon__External_ID__c",
+          "label": "Action",
+          "Filter Criteria": {
+              "is_child": true,
+              "First Level Parent Objects": [
+                  {
+                      "Object API Name": "clcommon__Actors__c",
+                      "Reference Field Name": "clcommon__Actor__c",
+                      "is_reference_mandatory": false
+                  }
+              ]
+          }
+      },
+      {
```

Comment:
Add this here.

{
        "Sequence": 4,
        "Object API Name": "clcommon__Portal_Layout__c",
        "External Id Field": "clcommon__External_ID__c ",
         "label": "Portal Layout",
        "Filter Criteria": {
            "is_root": true,
            "Where Clause": ""
        }
    },